pygfx/renderers/wgpu/engine/shared.py

```diff
@@ -63,6 +64,13 @@ def __init__(self, *, canvas=None):
         # Select adapter to use.
         if Shared._selected_adapter:
             self._adapter = Shared._selected_adapter
+        elif adapter_name := os.environ.get("PYGFX_WGPU_ADAPTER_NAME"):
+            # Similar to https://github.com/gfx-rs/wgpu?tab=readme-ov-file#environment-variables
```

Comment:
this begs the question: can't you just use `WGPU_ADAPTER_NAME`?

Reply:
i tried, it didnt work for me at the time.

i concluded that it was only for "testing and example infrastructure"